skyuxconfig.json

```diff
@@ -11,5 +11,10 @@
   ],
   "app": {
     "title": "Stache 2"
+  },
+  "appSettings": {
+    "stache": {
```

Comment:
It would probably be better to isolate the configuration under `"googleAnalytics"`. Also, the user should be able to enable/disable analytics manually (defaults to `true`). The use case would be that they wish to disable analytics for a short period of time, but maintain their credentials in the config.

```
{
  "stache": {
    "googleAnalytics": {
      "enabled": "true",
      "analyticsClientId": "",
      "tagManagerContainerId": ""
    }
  }
}
```